test/common/http_support/server.py

```diff
@@ -23,7 +23,7 @@ def __runServer(httpbinPort=0, httpPort=0, sslPort=0):
     # -- generate self signed SSL certificate
     
     privateKey = tempfile.NamedTemporaryFile(suffix='.pem')
-    subprocess.check_call(['openssl', 'genrsa'], stdout=privateKey, stderr=subprocess.PIPE)
+    subprocess.check_call(['openssl', 'genrsa', '2048'], stdout=privateKey, stderr=subprocess.PIPE)
```

Comment:
This test was failing with "key too short" so increasing key length to 2048 which is pretty standard. It doesn't slow down that much since RSA keys are pretty quick.